### PR TITLE
Add Redash to Analytics/BI

### PR DESCRIPTION
**Pull request by arikfr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,6 +106,7 @@ _Web Analytics_
 _Business Intelligence_
 
   * [Metabase](http://www.metabase.com/) - Simple Dashboarding and GUI Query tool, Nightly Emails and Slack Integration w/ PostgreSQL, MySQL, Redshift and other DBs. ([Source Code](https://github.com/metabase/metabase)) `AGPLv3` `Clojure`
+  * [Redash](http://redash.io) - connect to over 18 types of databases (SQL and "NoSQL"), query your data, visualize it and create dashboards. Everything has a URL that can be shared. Slack and HipChat integration. `BSD` `Python`
```
Comment (Kickball) [inline README.md]: This looks to be a paid application, it will need to be moved NON-FREE.MD.

Comment (arikfr) [inline README.md]: There is a paid hosted option ([to support the development of the project](https://medium.com/@arikfr/8e8259ac80cb)), but the project is open source, free and you can install it on your own. I even make it easy with [ready made images for AWS, GCE and Docker](http://docs.redash.io/en/latest/setup.html) (along with a setup script for other types of installation).

I probably need to improve the website if you missed that :-(

**Comment by Kickball** [comment]
Having another look over it I can now see that there is a free selfhosted option. 

Could you please add in a link to the demo and source code as per the contributing format?

`` [Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language` ``

**Comment by arikfr** [comment]
Done. Sorry for missing it earlier.

**Comment by Kickball** [comment]
Perfect, thanks for making those changes it should make it a lot more obvious that it isn't paid.

**Comment by arikfr** [comment]
I'll be happy to understand what on the website made you think it's a paid software only.
It's probably not for the comments here, so you're welcome to email me (arik at redash.io).

Thanks! (for making this list and everything)

**Pull request merged**